experimenter/experimenter/nimbus_ui_new/tests/test_forms.py

```diff
@@ -322,11 +327,93 @@ def test_unsubscribe_form_removes_subscriber(self):
         self.assertTrue(form.is_valid())
         form.save()
         self.assertNotIn(self.request.user, self.experiment.subscribers.all())
-        changelog = self.experiment.changes.get()
+        changelog = self.experiment.changes.latest("changed_on")
         self.assertEqual(changelog.changed_by, self.user)
         self.assertIn("dev@example.com removed subscriber", changelog.message)
 
 
+class TestLaunchForms(RequestFormTestCase):
+    def setUp(self):
+        super().setUp()
+        self.experiment = NimbusExperimentFactory.create()
+
+    def test_draft_to_preview_form(self):
```

Comment:
Oh all these tests need to setup the status/status_next/publish_status fields to some known values before we update them and assert on them, otherwise they could already be in those states, the form could do nothing, and they'd still pass 🙏